tests/components/plant/test_init.py

```diff
@@ -187,17 +188,17 @@ async def test_brightness_history(hass):
     assert await async_setup_component(
         hass, plant.DOMAIN, {plant.DOMAIN: {plant_name: GOOD_CONFIG}}
     )
-    hass.states.async_set(BRIGHTNESS_ENTITY, 100, {ATTR_UNIT_OF_MEASUREMENT: "lux"})
+    hass.states.async_set(BRIGHTNESS_ENTITY, 100, {ATTR_UNIT_OF_MEASUREMENT: LIGHT_LUX})
```

Comment:
This is a breaking change; unit of measurement changed from `lux` to `lx`, which can affect things like InfluxDB.